russian/vodka/vodka.md

```diff
@@ -1,4 +1,5 @@
 Живительная влага мягко разливается по измученному организму и невероятным
 усилием воли получается наконец открыть глаза.
 Глаза открыты, в руках топор.
-[Я делаю бревно.](Brevno/brevno.md)
\ No newline at end of file
+[Я делаю бревно.](Brevno/brevno.md)
+[Я звоню маме.](../call-mom/call-mom.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.